src/Landing/components/RecommendArticles.jsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { Row, Col, Container } from 'react-bootstrap';
+import Article from '../../commons/components/Article';
+import ErrorLabel from '../../commons/components/ErrorLabel';
+import '../CSS/RecommendedArticles.scss';
+import ActionButton from './ActionButton';
+
+function RecommendedArticles({ recommendedArticles, fetchStatus }) {
+  return (
+    <div className="recommended-articles">
+      <h2 className="section-title">Recommended Articles</h2>
+      {
+        fetchStatus
+          ? (
+            <Container>
+              <Row>
+                <Col sm={4} lg>
+                  <Article article={recommendedArticles[1]} />
+                </Col>
+                <Col sm={4} lg className="sm">
+                  <Article article={recommendedArticles[2]} />
+                  <Article article={recommendedArticles[3]} />
+                  <Article article={recommendedArticles[4]} />
+                </Col>
+                <Col sm={4} lg>
+                  <Article article={recommendedArticles[5]} />
+                </Col>
+              </Row>
+              <ActionButton />
+            </Container>
+          )
+          : (
+            <ErrorLabel
+              message="No Recommended Articles Fetched"
```

Comment:
Very good on this... my recommendation is that you start out by displaying a spinner and then depending on whether the articles were fetched or not you display the appropriate content i.e. either the articles or the error message

Reply:
@AGMETEOR kindly note that this has been implemented.